src/backend/Csrs.Api/Models/ModelExtensions.cs

```diff
@@ -72,15 +70,44 @@ public static MicrosoftDynamicsCRMssgCsrsparty ToDynamicsModel(this Party party)
                 //SsgPortalaccess = ssgPortalaccess, ????
                 SsgIdentity = party.Identity?.Id,
                 //SsgIdentityotherdetails = ssgIdentityotherdetails, ???
-                SsgStreet1 = party.AddressStreet1
+                SsgStreet1 = party.AddressStreet1,
+                SsgIncomeassistance = party.IncomeAssistance
             };
             return dynamicsParty;
         }
 
-        public static async Task<MicrosoftDynamicsCRMssgCsrsfile> ToDynamicsModel(this File file, IDynamicsClient dynamicsClient, CancellationToken cancellationToken)
+
+        public static async Task<File> ToViewModelAsync(MicrosoftDynamicsCRMssgCsrsfile dynamicsFile, IDynamicsClient dynamicsClient, IMemoryCache cache, CancellationToken cancellationToken)
+        {
+            return null;
+        }
+
+        private static int? GetSection7Expenses(string? value)
+        {
+            if (value is null)
+            {
+                return null;
+            }
+            return value.Equals("Yes") ? (int)Section7Expenses.Yes :
+                   value.Equals("No") ? (int)Section7Expenses.No :
+                                         (int)Section7Expenses.IDontKnow;
+        }
+
+        private static int? GetPartyEnrolled(string? value)
+        {
+            if (value is null) return null;
+            return value.Equals("Recipient") ? (int)PartyEnrolled.Recipient : (int)PartyEnrolled.Payor;
+        }
+
+        private static bool? ConvertToBool(string? value)
```

Comment:
already cleaned up
![image](https://user-images.githubusercontent.com/1844480/153967108-2438f0a9-1fe6-4a52-9189-884195e49d4a.png)

Reply:
please resolve this after this PR has been merged.